guide/english/documentation/index.md

```diff
@@ -3,14 +3,19 @@ title: Code documentation
 ---
 ## Code Documentation
 
-Code documentation is a way developers write their code to create the best versions of their functions possible. Code Documentation lets the newbie to get comfortable with the customs of the particular module, functions etc for a particular programming language. It is always recommended to go with the documentation before debugging your code, this helps you to debug effectively.
+Code documentation is a way in which developers write their code to create the best versions of their functions possible. Code Documentation lets the newbie to get comfortable with the customs of the particular module, functions etc for a particular programming language. It is always recommended to go with the documentation before debugging your code, this helps you to debug effectively.
```

Comment:
Please change "Code Documentation lets the newbie to get comfortable with..." to "Code Documentation lets the newbie get comfortable with....". Basically just omit the "to".